src/auth.ts

```diff
@@ -4,21 +4,35 @@ import {
   IRouter
 } from '@jupyterlab/application';
 
-import { AuthWidget } from './widget';
-
 import { reactIcon } from '@jupyterlab/ui-components';
 
+import { getRandomColor } from '@jupyterlab/docprovider/lib/awareness';
```

Comment:
We need to export awareness [here](https://github.com/jupyterlab/jupyterlab/blob/99b9cbc9fee00a392893b74764c66207bcc399b9/packages/docprovider/src/index.ts#L10-L12) since is not public for extensions.